.github/workflows/docker.yml

```diff
@@ -0,0 +1,140 @@
+name: Docker unstable release
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  ssr:
+    name: Build SSR images
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Get current date
+        id: date
+        run: echo "::set-output name=date::$(date +'%Y-%m-%d')"
+
+      - name: Get commit hash
+        run: echo "SHORT_SHA=${GITHUB_SHA::8}" >> $GITHUB_ENV
```

Comment:
mind me asking why you expose the values in 2 different ways here?
once as a step output (date) and the other as a env (short_sha)

Works perfectly fine just noticed that and was wondering why

Reply:
At first, this step action was here for creating the appropiate tag based on the release type, and I couldn't have duplicated IDs, so I needed to use env vars. Finally, it seemed cleaner to me to decouple it in another file, so just took the original approach, changed the name and leave it! :D

I can revisit this in another PR where I will integrate this with GH Container Registry.

Reply:
fair enough, was just curious and can definitely be revised on later :wink: